lib/spack/env/cc

```diff
@@ -21,6 +21,25 @@
 #      -Wl,-rpath         arguments for dependency /lib directories.
 #
 
+if [ -n "$ZSH_VERSION" ]; then
+    # ensure posix-shell compatibility for field splitting
+    emulate -R sh
+    unset IFS
+    load_prof() {
+        emulate -L zsh
+        zmodload zsh/zprof
+        echo prof loaded
+    }
+    print_prof() {
+        emulate -L zsh
+        zprof >opt.txt
+        echo prof printing
+    }
+    if [[ $PROF_CC == "true" ]]; then
+        load_prof
+    fi
```

Comment:
Are these just leftover debug infrastructure from working on this PR?

Reply:
The emulation is not, the PROF stuff is profiling, it does not need to be there persistently.